xml/containers_basics.xml

```diff
@@ -4,83 +4,75 @@
+  with kernel Control Groups (<xref linkend="gloss-vt-lxc-cgroups"/>) and
+  kernel Namespaces.
  </para>
  <para>
-  Containers provide virtualization at the operating system level where the
-  <emphasis>kernel</emphasis> controls the isolated containers.
-  This is unlike full virtualization solutions like &xen; or &kvm; where the
-  <emphasis>processor</emphasis> simulates a complete hardware environment
-  and controls virtual machines.
+  Containers provide virtualization at the operating system level, where the
+  <emphasis>kernel</emphasis> controls the isolated containers. This approach
+  differs from full virtualization solutions like &xen; or &kvm;, where the
+  <emphasis>processor</emphasis> simulates a complete hardware environment and
+  controls virtual machines.
  </para>
-
- <!--<sect1 xml:id="sec-containers-intro">
+<!--<sect1 xml:id="sec-containers-intro">
   <title>Overview</title>-->
-
-  <para>
-   Conceptually, containers can be seen as an improved
-   <xref linkend="gloss-vt-lxc-chroot"/> technique. The difference is that a
-   chroot environment separates only the file
-   system, whereas containers go further and provide resource management
-   and control via <xref linkend="gloss-vt-lxc-cgroups"/>.
-  </para>
-
-  <itemizedlist mark="bullet" spacing="normal">
-   <title>Benefits of Containers</title>
-   <listitem>
-    <para>
-     Isolating applications and operating systems through containers.
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     Providing nearly native performance as container manages allocation of
-     resources in real-time.
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     Controlling network interfaces and applying resources inside containers
-     through <xref linkend="gloss-vt-lxc-cgroups"/>.
-    </para>
-   </listitem>
-  </itemizedlist>
-
-  <itemizedlist mark="bullet" spacing="normal">
-   <title>Limitations of Containers</title>
-   <listitem>
-    <para>
-     All containers run inside the host system's kernel and not with a
-     different kernel.
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     Only allows Linux <quote>guest</quote> operating systems.
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     Security depends on the host system. Container is not secure. If you
-     need a secure system, you can confine it using an &aa; or &selnx;
-     profile.
-    </para>
-   </listitem>
-  </itemizedlist>
- <!--</sect1>-->
+ <para>
+  Conceptually, containers can be seen as an improved
+  <xref linkend="gloss-vt-lxc-chroot"/> technique. While a chroot environment
+  separates only the file system, containers go further and provide resource
+  management and control via <xref linkend="gloss-vt-lxc-cgroups"/>.
+ </para>
+ <itemizedlist mark="bullet" spacing="normal">
+  <title>Advantages of Using Containers</title>
+  <listitem>
+   <para>
+    Isolate applications and operating systems through self-contained units.
```

Comment:
It is not possible to run different operating systems in containers. All share the same kernel as the host. 

The closest thing is the ability to run a different Linux distribution in a container, but on the same kernel. 

This is an important distinction.

Reply:
What does this comment refer to? The fact that containers use the same kernel as the host is mentioned in the "Limitations" section.

Reply:
Yes, understood, I saw that -- but this line claims the ability to isolate operating systems. I feel that is a misleading claim.

I am being picky about this for 2 reasons -- [1] there is a lot of confusion surrounding containers and what they do, and they're often confused with VMs and hypervisors, and [2] because in the latest tech, the situation is actually getting _more_ confusing, because tools like Ubuntu's LXD, Google GVisor, Intel ClearContainers and Clear Linux, etc. are blurring the lines between VMs and containers. 

Therefore I feel it is important to be as unambiguous as possible.